cmd/livepeer/livepeer.go

```diff
@@ -408,10 +408,21 @@ func stream(port string, streamID string) {
 		return
 	}
 
+	//Fetch local stream playlist - this will request from the network so we know the stream is here
 	url := fmt.Sprintf("http://localhost:%v/stream/%v.m3u8", port, streamID)
+	res, err := http.Get(url)
+	if err != nil {
+		glog.Fatal(err)
+	}
+	_, err = ioutil.ReadAll(res.Body)
+	res.Body.Close()
+	if err != nil {
+		glog.Fatal(err)
+	}
+
 	cmd := exec.Command("ffplay", "-timeout", "180000000", url) //timeout in 3 mins
```

Comment:
Yeah - res.Body contains the actual content of the playlist.  The url is what we want.